apps/vanilla/src/components/api.ts

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios';
+import { Pagination } from '@js-camp/core/models/pagination';
+import { PaginationMapper } from '@js-camp/core/mappers/pagination.mapper';
+import { AnimeType } from '@js-camp/core/enums/animeType';
+import { Anime } from '@js-camp/core/models/anime';
+
+/** A class designed to interact with API. */
+export class API {
+  public constructor(
+    private readonly baseUrl: string,
+    public urlQuery: string,
+  ) {
+  }
+
+  /**
+   * Send request.
+   * @param url Link.
+   */
+  public async getPagination(url: string): Promise<Pagination<Anime>> {
+    const response = await axios.get(url);
```

Comment:
Pass generic into `get`, otherwise `response` has `any` type
```suggestion
    const response = await axios.get<PaginationDto<AnimeDto>>(url);
```

Reply:
Corrected